### ISSUE TITLE
Graphical interface

### ISSUE THREAD

**Issue by lexis33** [101dotxyz/GPTeam#76]
Would be great to see 3d or 2d world where you can see the movement of the Characters. Might be kind of game or world simulation.

**Comment by joshsny** [comment]
@lexis33 A better interface is in the pipeline 👀

**Comment by inferont** [comment]
@joshsny It's been about a month. Do we have designs for this interface?

**Comment by joshsny** [comment]
Yep, now live at GPTeamAI.comYou can try it out completely free without even needing an API key 🔑On 21 Jul 2023, at 22:53, Ryan ***@***.***> wrote:﻿
@joshsny It's been about a month. Do we have designs for this interface?

**Comment by inferont** [comment]
@joshsny That looks like a significant fork of this codebase. Is there a considerable merge happening in the future? This project has great potential, and I would like to contribute. I have a social science/philosophy background and 15+ years of experience in the software industry.
